StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -82,6 +82,7 @@ extension EmbeddedPaymentElement {
             shouldShowMandate: configuration.embeddedViewDisplaysMandateText,
             savedPaymentMethods: loadResult.savedPaymentMethods,
             customer: configuration.customer,
+            incentive: loadResult.elementsSession.linkSettings?.linkConsumerIncentive.flatMap { PaymentMethodIncentive(from: $0) },
```

Comment:
Just curious, why `flatMap` here? 

Side note, if you want to be extra Swifty, I believe this could be `.flatMap(PaymentMethodIncentive.init)` 😎

Reply:
`flatMap` to deal with nullability.